crates/uv-distribution/src/metadata/lowering.rs

```diff
@@ -33,25 +35,51 @@ enum RequirementOrigin {
 
 impl LoweredRequirement {
     /// Combine `project.dependencies` or `project.optional-dependencies` with `tool.uv.sources`.
+    #[allow(clippy::needless_pass_by_value)]
     pub(crate) fn from_requirement<'data>(
         requirement: uv_pep508::Requirement<VerbatimParsedUrl>,
         project_name: &'data PackageName,
         project_dir: &'data Path,
         project_sources: &'data BTreeMap<PackageName, Sources>,
         project_indexes: &'data [Index],
+        extra: Option<ExtraName>,
+        group: Option<GroupName>,
```

Comment:
Should these be `Option<&ExtraName>` and `Option<&GroupName>`? It looks like the Clippy lint for this was suppressed explicitly, so I'm guessing there's a reason for it, but it doesn't immediately stand out to me.

Reply:
They should but I can't for the life of me figure out how to get it to work at the call site.

Reply:
Figured it out by removing a lifetime.

Reply:
Ahhh yeah I see now. Nice.